decidim/decidim
# Move social login buttons to the top of the login modal

#### :tophat: What? Why?

While seeing about #8465, I've seen that we actually have different behavior between the login page and the login modal regarding the OAuth providers.

In the login page, we have the full buttons (icon + text) and they're on the top of the page. 
In the login modal, we have the icon button, and they're on the bottom. 

This PR fixes that, having the same behavior between these two sections. 

As this touches design, this needs to be reviewed by @carolromero 

#### :pushpin: Related Issues

- Fixes #8465 
- Related to #4457

#### Testing

1. Go to an action that needs a login as a visitor, for instance a "Follow" button
2. Click on it
3. See modal

### :camera: Screenshots

Bear in mind that this is like the worst case scenario with 5 providers. I know instances with maximum of 3 OAuth providers, so it shouldn't break in mobile. 

|Before|After| 
|-----|----|
|![image](https://user-images.githubusercontent.com/717367/144045644-3ca96f1f-d056-478d-b687-e726be6a6b36.png) |![image](https://user-images.githubusercontent.com/717367/144045560-47449dd5-d7d2-4681-ab99-99ef37056462.png)|

:hearts: Thank you!
